Doc/library/faulthandler.rst

```diff
@@ -81,6 +93,10 @@ Fault handler state
    The *file* must be kept open until the fault handler is disabled: see
    :ref:`issue with file descriptors <faulthandler-fd>`.
 
+   If *c_stack* is ``True``, then the C stack trace is printed after the Python
+   traceback, unless the system doesn't support it. See :func:`dump_c_stack` for
```

Comment:
```suggestion
   traceback, unless the system does not support it. See :func:`dump_c_stack` for
```

I'm not sure we use short forms in this file or not, just check it (I don't have an IDE opened).